nss-ivory-trolls/BangazonAPI
# Ac department controller

# Description
Feature adds CRUD functionality to Department Controller. Added automated testing to Department Controller.

## Type of change

# Testing Instructions
1. git checkout ac-department-controller
1. dotnet run
1. open PostMan

Ensure the following HTTP requests are functional:
- Get (All)
- Get (Id)
- Post
- Put

1. Navigate to TestBangazonAPI project. Ensure that all tests in DepartmentCRU file pass. 

# Checklist:
